packages/tools/fetch-tool/src/fluidFetchArgs.ts

```diff
@@ -121,6 +121,7 @@ export function parseArguments() {
             case "--help":
                 printUsage();
                 process.exit(0);
+            // fallthrough
```

Comment:
optional nit: I'd rather see this fixed by putting `break;` in the previous case.